packages/tree-widget/package.json

```diff
@@ -0,0 +1,97 @@
+    "build": " tsc 1>&2 && npm run copy:assets && npm run pseudolocalize",
+    "clean": "rimraf lib",
+    "lint": "tslint -p . 1>&2",
+    "test": "",
+    "cover": ""
+  },
+  "keywords": [
+    "Bentley",
+    "iModel Tree Widget",
+    "iModel.js"
+  ],
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/tree-widget"
+  },
+  "peerDependencies": {
+    "@bentley/bentleyjs-core": "2.2.0",
+    "@bentley/reality-data": "3.5.4",
+    "@bentley/context-registry-client": "2.2.0",
+    "@bentley/frontend-authorization-client": "2.2.0",
+    "@bentley/geometry-core": "2.2.0",
+    "@bentley/imodelhub-client": "2.2.0",
+    "@bentley/imodeljs-common": "2.2.0",
+    "@bentley/imodeljs-frontend": "2.2.0",
+    "@bentley/imodeljs-i18n": "2.2.0",
+    "@bentley/imodeljs-markup": "2.2.0",
+    "@bentley/imodeljs-quantity": "2.2.0",
+    "@bentley/itwin-client": "2.2.0",
+    "@bentley/orbitgt-core": "2.2.0",
+    "@bentley/presentation-common": "2.2.0",
+    "@bentley/presentation-frontend": "2.2.0",
+    "@bentley/product-settings-client": "2.2.0",
+    "@bentley/ui-abstract": "2.2.0",
+    "@bentley/ui-components": "2.2.0",
+    "@bentley/ui-core": "2.2.0",
+    "@bentley/ui-framework": "2.2.0",
+    "@bentley/ui-ninezone": "2.2.0",
+    "@bentley/webgl-compatibility": "2.2.0"
+  },
+  "devDependencies": {
+    "@bentley/bentleyjs-core": "2.2.0",
+    "@bentley/build-tools": "2.2.0",
+    "@bentley/context-registry-client": "2.2.0",
+    "@bentley/frontend-authorization-client": "2.2.0",
+    "@bentley/geometry-core": "2.2.0",
+    "@bentley/imodelhub-client": "2.2.0",
+    "@bentley/imodeljs-common": "2.2.0",
+    "@bentley/imodeljs-frontend": "2.2.0",
+    "@bentley/imodeljs-i18n": "2.2.0",
+    "@bentley/imodeljs-markup": "2.2.0",
+    "@bentley/imodeljs-quantity": "2.2.0",
+    "@bentley/itwin-client": "2.2.0",
+    "@bentley/orbitgt-core": "2.2.0",
+    "@bentley/presentation-common": "2.2.0",
+    "@bentley/presentation-frontend": "2.2.0",
+    "@bentley/product-settings-client": "2.2.0",
+    "@bentley/react-scripts": "3.4.1",
+    "@bentley/reality-data": "^3.5.4",
+    "@bentley/ui-abstract": "2.2.0",
+    "@bentley/ui-components": "2.2.0",
+    "@bentley/ui-core": "2.2.0",
+    "@bentley/ui-framework": "2.2.0",
+    "@bentley/ui-ninezone": "2.2.0",
+    "@bentley/webgl-compatibility": "2.2.0",
+    "@types/classnames": "^2.2.3",
+    "@types/i18next": "^8.4.2",
+    "@types/react": "~16.9.35",
+    "@types/react-dom": "^16.8.0",
+    "@types/react-redux": "^7.0.1",
+    "cpx": "^1.5.0",
+    "rimraf": "^3.0.2",
+    "tslint": "^5.11.0",
+    "typescript": "~3.7.4"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "i18next": "^10.2.2",
+    "react": "^16.8.0",
```

Comment:
@calebmshafer  should `react` and `react-dom` be a dev+peer Dep instead?

Reply:
Yes, they probably should be.  That's the way we've moved for the iModel.js packages to avoid pulling multiple versions of React, which causes issues when hooks.

Reply:
fixed